terraform/roar_variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "mix_validators" {
+  type    = bool
+  default = false
+  description = "Mix validator images for ROAR test"
+}
+
+variable "mix_image_tags" {
+  type        = list(string)
+  default     = ["latest_dynamic"]
+  description = "List of Docker image tags to be used in ROAR test"
```

Comment:
lol "ROAR" probably doesn't make any sense to many people. Let's be more descriptive here by using the full name

Reply:
aw man roar sounds cooler 8) but ok sure thing